src/main/resources/config/liquibase/changelog/20220328211300_changelog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="mallgaier" id="20220328211300">
+        <addColumn tableName="exam">
+            <column name="test_exam" defaultValueBoolean="false" type="boolean"/>
+            <column name="working_time" type="INT">
+                <constraints nullable="false"/>
```

Comment:
Is there a specific reason why `working_time` is not nullable?

Reply:
The `TestExams` have made the attribute `workingTime` necessary. So far, the working time is calculated from the start and end date. To keep continuity, liquibase-changelog is used to calculate and insert the working time for all previous exams. 
For future `RealExams` the working time will be calculated in the client / server side and saved. Therefore, the decision was made to set this as a non-null attribute. 
Furthermore, it would be possible in the future, for example, to use the attribute `workingTime` for `RealExams`, instead of calculating this each time from the start and end time.